### PR TITLE
Update subscriptions serializers with site's default currency and currency code

### PR DESCRIPTION
**Pull request by asadmanzoor93**
**Description:**  This PR updates subscriptions serializers with the site's default currency and currency code.

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-3640
https://edlyio.atlassian.net/browse/EDLY-3616

**Merge checklist:**

- [ ] All reviewers approved
- [ ] Commits are (reasonably) squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Note:** Please add screenshots for any viewable change.
screenshot-dev-payments-multisitesdev-edly-io-api-v2-subscriptions-1631277101695

### CODE REVIEW

**Review by burhandodhy — CHANGES_REQUESTED** [review]
Kindly update tests.

**Review by burhandodhy — APPROVED** [review]
LGTM

File: ecommerce/subscriptions/api/v2/tests/test_serializers.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Unit tests for subscription API serializers.
 """
 import ddt
-
+from django.conf import settings
 from ecommerce.subscriptions.api.v2.serializers import (
```
Comment (taimoor-ahmed-1) [inline ecommerce/subscriptions/api/v2/tests/test_serializers.py:6]: add empty line

Comment (burhandodhy) [reply ecommerce/subscriptions/api/v2/tests/test_serializers.py]: Done

**Review by taimoor-ahmed-1 — APPROVED** [review]
👍

**Pull request merged**